### ISSUE TITLE
Powershell Issues

### ISSUE THREAD

**Issue by Worble** [1MB/1mb-cli#1]
Printing `1mb` into powershell will simply return the amount of bytes in 1MB, i.e
```
deploy> 1mb
1048576
```
In order to run the cli, a \` must be placed beforehand like so

```
deploy> `1mb
Quickly deploy static content to 1MB's free static hosting service
```

Took me a while to track this issue (as an aside, the website states the cli binary name is 1mb-cli). I assume the intention is to eventually change it to 1mb-cli?

**Comment by 0x15f** [comment]
That's very strange, thank you for debugging @Worble. Yes, in the next big release `2.0.0` coming in a few days the command will be changed to `1mb-cli`.

**Comment by 0x15f** [comment]
This will be fixed in PR #3
